File: Cetiri.java
import java.util.Arrays;
import java.util.Scanner;

public class Cetiri {
	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
		int arr[] = new int[3];
		arr[0] = sc.nextInt();
		arr[1] = sc.nextInt();
		arr[2] = sc.nextInt();
		
		Arrays.sort(arr);
		
		int difOne = Math.abs(arr[0]-arr[1]);
		int difTwo = Math.abs(arr[1]-arr[2]);
		
		if(difOne == difTwo) {
			System.out.println(difTwo+arr[2]);
		} else if(difOne / 2 == difTwo) {
			System.out.println(arr[0]+difTwo);
		} else if(difTwo / 2 == difOne) {
			System.out.println(arr[1] + difOne);
		}
		
	}

}

File: Python Solutions/Cetiri.py
arr = list(map(int,input().split()))
arr.sort()
if(abs(arr[0]-arr[1]) == abs(arr[1]-arr[2])):
    print(arr[2]+abs(arr[0]-arr[1]))
elif(abs(arr[0]-arr[1]) > abs(arr[1]-arr[2])):
    print(arr[0] + abs(arr[1]-arr[2]))
else:
    print(arr[1] + abs(arr[0]-arr[1]))